core/src/main/scala/java/nio/Foo.scala

```diff
@@ -0,0 +1,2 @@
+package java.nio.file {
```

Comment:
you can put this in the `scala-2.10` directory only if you want to limit the hackiness.